pay-api/requirements/repo-libraries.txt

```diff
@@ -1 +1,2 @@
 -e git+https://github.com/bcgov/sbc-common-components.git#egg=sbc-common-components&subdirectory=python
+git+https://github.com/daxiom/simple-cloudevent.py.git@0.0.2
```

Comment:
bcgov repo?

Reply:
Following the same pattern as Patrick, with the local repo to start

Reply:
Ok, as long as this gets cleaned up later